### DISCUSSION TITLE
Error please solve

### DISCUSSION THREAD

**Discussion by Shresth-Sharma** [community/community#21420] [Other Feature Feedback, Questions, & Ideas]
<p>please tell me the sollution why output is blank</p>

**Comment by Shresth-Sharma** [comment]
<pre><code class="lang-auto">[type or paste code here](https://shresth-sharma.github.io/t/)
</code></pre>

**Comment by jfullstackdev** [comment, marked as answer]
<p>you have one error at this point, it cannot load <code>lose.mp3</code>, it should be <code>Lose.mp3</code></p>
<p><img src="https://user-images.githubusercontent.com/47092464/181064501-a7d0a74f-d276-45b6-bd90-5cb47352f432.png" alt="image" data-base62-sha1="ybYKfrwt4t3UcQZUTLYcSavM3QB" width="533" height="226"></p>
<p>if I type <code>Lose.mp3</code>, it’s the correct one, now it’s loading and playing</p>
<p><a href="https://shresth-sharma.github.io/t/Lose.mp3">https://shresth-sharma.github.io/t/Lose.mp3</a></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/a/d/ad2e331a3243e082a92a7d0c74dddd93ca1c1c82.png" data-download-href="/uploads/short-url/oI1AffPrKIGAtvp3Cdov5NoB4em.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181064502-5725654a-6ec4-450d-ab52-d8b2ec07d717.png" alt="image" data-base62-sha1="oI1AffPrKIGAtvp3Cdov5NoB4em" width="690" height="373" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/a/d/ad2e331a3243e082a92a7d0c74dddd93ca1c1c82_2_690x373.png, https://aws1.discourse-cdn.com/github/optimized/3X/a/d/ad2e331a3243e082a92a7d0c74dddd93ca1c1c82_2_1035x559.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/a/d/ad2e331a3243e082a92a7d0c74dddd93ca1c1c82_2_1380x746.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/a/d/ad2e331a3243e082a92a7d0c74dddd93ca1c1c82_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1040 19.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
